src/alks.ts

```diff
@@ -152,12 +152,15 @@ namespace ALKS {
    * @property denyArns - The ARNs for the deny policies associated with this role
    * @property instanceProfileArn - The Instance Profile ARN associated with this role
    * @property addedRoleToInstanceProfile - Whether this role was added to an Instance Profile
+   * @property RoleTags - The tags associated with the IAM account role
```

Comment:
Shouldnt this just be called 'tags' since thats what you named it in the interface?

Reply:
I forgot to change it when I changed the object name. I didn't run typedoc so this is localized within the source code